Automattic/jetpack
# Add broadcast channel listener to ensure login messages are received

This PR sets up verbum to listen to the broadcasting channel created here: D138915-code
Users with passwordless login can open the login link in a new tab, and in this case `parent.opener` is not available and we can't post the verbum message.
The listener on the channel reuses the same logic as before, but now we ensure that the popup we open for login is always closed after the message is received.

Fixes #

## Proposed changes:

* Listen to the broadcast channel

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync this PR
* Sync D138915-code as well
* Open a post and login with passwordless user
* The login popup should be closed, and the user successfully logged in.
* Test the login by clicking on the email link and by pasting the link in a new tab.
